### PR TITLE
chore: #2402 ui updates tweaks to the developers portal

### PR DESCRIPTION
**Pull request by phmngocnghia**
# Pull request checklist

## Does this close any currently open issues?

fixes: #2402 

**Please check if your PR fulfills the following requirements:**

- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build (`yarn build`) was run locally and any changes were pushed
- [x] Lint (`yarn lint`) has passed locally and any fixes were made for failures
- [x] Test (`yarn test`) has passed locally and any fixes were made for failures

## Pull request type

**Please check the type of change your PR introduces:**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

- Register
image

- Auth Type
image

- Webhooks
image

- Filter
image

- Copy
image

- Billing Page
image

Issue Number: #

## What is the new behavior?

- Register
image

- Auth Type
image

- Webhooks
image

- Filter
image

- Copy
image

- Billing Page
image

-

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: packages/developer-portal/src/components/pages/webhooks/webhooks.tsx [hunk 1]
```diff
@@ -204,7 +204,7 @@ export const DeveloperWebhooks = () => {
         <FormSubHeading>
           This system is designed to flexibly work with how your application is built and deployed. If you wish, you can
           set up a single endpoint to catch all topics for all customers. Alternatively, you may wish to set up a
-          different webhook subscription per topic or per customer. For more information about Webhooks, please see our
+          different webhook subscription per topic or per customer. For more information about Webhooks, please see our{' '}
```
Comment (tanphamhaiduong) [inline packages/developer-portal/src/components/pages/webhooks/webhooks.tsx:207]: pls remove `{' '}`

Comment (phmngocnghia) [reply packages/developer-portal/src/components/pages/webhooks/webhooks.tsx:207]: Hi @tanphamhaiduong, `{' '} is placed intentionally to add spacing between webhooks link and paragraph

Comment (tanphamhaiduong) [reply packages/developer-portal/src/components/pages/webhooks/webhooks.tsx:207]: @phmngocnghia should we change to `&nbsp;` ?

**Review by vuhuucuong — APPROVED** [review]
LGTM

**Pull request merged**